Assets/00_Content/Scripts/Interactables/Beers/BeerTap.cs

```diff
@@ -141,6 +146,8 @@ public class BeerTap : Interactable {
 			fillProgressBar.UpdateProgress(fillPortion * beerAmount);
 			pourProgressBar.Hide();
 			perfectProgressIndicator.gameObject.SetActive(false);
+
+			pourSoundHandle?.FadeOutAndStop(0.2f);
```

Comment:
This seems to be done twice, here and when the coroutine ends. Is it supposed to be so?